src/aria/popups/PopupNavigationManager.js

```diff
@@ -433,5 +642,236 @@ function ViewportNavigationInterceptor() {
+        ariaUtilsArray.forEach,
+        ariaUtilsArray,
+        reverters,
+        ariaUtilsFunction.call
+    );
+
+    // ------------------------------------------------------------------ return
+
+    return reverter;
+};
+
+/**
+ * Show back all elements around the given element, by applying the same traversal as for hideOthers.
+ *
+ * @param {HTMLElement} element The reference element
+ */
+prototype.showOthers = function(element) {
+    var self = this;
+    traverseSiblingsUntilRootOfBranch(element, function(element) {
+        self.show(element);
+    });
+};
+
+
+/**
+ * Registers the request of hiding the given element, and actually hides it if this is the first one.
+ *
+ * <p>
+ * The current implementation uses an attribute "data-hide-requests-count" to store the number of requests.
+ * </p>
+ *
+ * @param {HTMLElement} element The element to hide
+ *
+ * @return {Boolean} Whether or not the element was actually hidden (first request)
+ */
+prototype.hide = function(element) {
+    // ----------------------------------------------------------- destructuring
+
+    var attributeName = this.attributeName;
+
+    var hideRequestsCount = element.getAttribute(attributeName);
+    if (hideRequestsCount == null) {
+        hideRequestsCount = 0;
+    }
+
+    // -------------------------------------------------------------- processing
+
+    var hidden = false;
+
+    if (hideRequestsCount === 0) {
+        element.setAttribute('aria-hidden', 'true');
+        hidden = true;
+    }
+
+    hideRequestsCount++;
+
+    // --------------------------------------------------------- output & return
+
+    element.setAttribute(attributeName, hideRequestsCount);
+
+    return hidden;
+};
+
+/**
+ * Registers the request of showing the given element by decreasing the number of hiding requests, and actually shows it back if there are no more such requests.
+ *
+ * <p>
+ * The current implementation uses an attribute "data-hide-requests-count" to store the number of requests.
+ * </p>
+ *
+ * @param {HTMLElement} element The element to show
+ *
+ * @return {Boolean} Whether or not the element was actually shown (when no more hiding requests)
+ */
+prototype.show = function(element) {
+    // ----------------------------------------------------------- destructuring
+
+    var attributeName = this.attributeName;
+
+    var hideRequestsCount = element.getAttribute(attributeName);
+
+    // ------------------------------------------------------- early termination
+
+    if (hideRequestsCount == null) {
+        return false;
+    }
+
+    // -------------------------------------------------------------- processing
+
+    var showed = false;
+
+    hideRequestsCount--;
+
+    if (hideRequestsCount === 0) {
+        element.removeAttribute('aria-hidden', 'true');
+        showed = true;
+    }
+
+    // --------------------------------------------------------- output & return
+
+    if (hideRequestsCount === 0) {
```

Comment:
For me they were a bit semantically distinct, hence the fact I did not merge them, but I can do it.